src/test/java/core/basesyntax/files/CsvFileWriterTest.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax.files;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class CsvFileWriterTest {
+    private static FileWriter fileWriter;
+    private static final String PATH = "src/test/resourcesTest/outputFile.csv";
+    private static final String REPORT = "fruit,quantity" + System.lineSeparator()
+            + "banana,152" + System.lineSeparator()
+            + "apple,90" + System.lineSeparator();
+
+    @Test
+    void writeToFile_correctPath_Ok() {
+        fileWriter = new CsvFileWriter();
```

Comment:
it should be in a method with annotation @Before

Reply:
not fixed